nilearn/input_data/nifti_labels_masker.py

```diff
@@ -59,6 +59,16 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
     standardize: boolean, optional
         If standardize is True, the time-series are centered and normed:
         their mean is put to 0 and their variance to 1 in the time dimension.
+        The ``standardize`` parameter is deprecated and will be removed, use
+        ``normalize="std"`` to standardize.
+
+    normalize: {"psc", "std", None}, optional
+        The time-series normalization method.
+        If 'psc' (percent signal change), input time-series means in the time
+        dimension are put to 100 prior to any temporal preprocessing.
+        If 'std', the output time-series are centered and normed:
+        their mean is put to 0 and their variance to 1 in the time dimension.
+        If None, no normalization is done.
```

Comment:
Same comment on the docstrings: remove standardize, and change the wording in normalize.